BaragonCore/src/main/java/com/hubspot/baragon/models/AgentRequestType.java

```diff
@@ -1,5 +1,5 @@
 package com.hubspot.baragon.models;
 
 public enum AgentRequestType {
-  APPLY, REVERT, CANCEL
+  APPLY, REVERT, CANCEL, PURGE_CACHE
```

Comment:
Ah, these must be where the json serialization issue is coming from, since older clients don't know about PURGE_CACHE yet